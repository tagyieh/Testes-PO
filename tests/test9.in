4
3
969011
8
969014
Vy9HwqXDY4RxlLFD8Ow02ef76KfFqrNKUuMfBRiuuKbIuUIVAIteoUwkhNwdCL3vAdiSXEW3X2mPMRCjwGmlA8c3JmEG9MGywcOooM5RQ51l4fVy9HwqXDY4RxlLFD8Ow02ef76KfFqrNKUuMfBRiuuKbIuUIVAIteoUwkhNwdCL3vAdiSXEW3X2mPMRCjwGmlA8c3JmEG9MGywcOooM5RQ51l4fVy9HwqXDY4RxlLFD8Ow02ef76KfFqrNKUuMfBRiuuKbIuUIVAIteoUwkhNwdCL3vAdiSXEW3X2mPMRCjwGmlA8c3JmEG9MGywcOooM5RQ51l4f
7
6
1
8
969014
OQTTQDukamlsNca
7
6
2
8
969015
OQTTQDukamlsNca
7
6
3
8
969014
98dHQUnrXXUCBQX8fxlvBtuTxGZHyAahheWQkV7YTD31HoUToc9eZNVV
7
6
4
8
969015
98dHQUnrXXUCBQX8fxlvBtuTxGZHyAahheWQkV7YTD31HoUToc9eZNVV
7
6
5
8
969014
30MqiAzIvOEaWas1IjhWlNBUD6ChxoBcCBQwOoYOr8Tyt960yTNG0SVZzYLsOZDNT1Ym5rnABu8B9tnn1BFwGjH0nQ0wUqTwyHoGuNl
7
6
6
8
969015
30MqiAzIvOEaWas1IjhWlNBUD6ChxoBcCBQwOoYOr8Tyt960yTNG0SVZzYLsOZDNT1Ym5rnABu8B9tnn1BFwGjH0nQ0wUqTwyHoGuNl
7
0
0
0
