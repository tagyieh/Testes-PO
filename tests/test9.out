MENU PRINCIPAL
1 - Abrir
2 - Guardar
3 - Gestão de clientes
4 - Gestão de terminais
5 - Menu de consultas
6 - Ver saldo global
0 - Sair
Escolha uma opção: GESTÃO DE TERMINAIS
1 - Mostrar todos os terminais
2 - Registar terminal
3 - Operações de terminal
0 - Sair
Escolha uma opção: Número do terminal: OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número do terminal: Mensagem de texto a enviar: OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Valores para o terminal '969011':0 (pagamentos),  660 (dívidas).
OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número da comunicação: OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número do terminal: Mensagem de texto a enviar: OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Valores para o terminal '969011':660 (pagamentos),  10 (dívidas).
OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número da comunicação: OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número do terminal: Mensagem de texto a enviar: OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Valores para o terminal '969011':670 (pagamentos),  10 (dívidas).
OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número da comunicação: OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número do terminal: Mensagem de texto a enviar: OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Valores para o terminal '969011':680 (pagamentos),  10 (dívidas).
OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número da comunicação: OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número do terminal: Mensagem de texto a enviar: OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Valores para o terminal '969011':690 (pagamentos),  10 (dívidas).
OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número da comunicação: OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número do terminal: Mensagem de texto a enviar: OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Valores para o terminal '969011':700 (pagamentos),  206 (dívidas).
OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número da comunicação: OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número do terminal: Mensagem de texto a enviar: OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Valores para o terminal '969011':906 (pagamentos),  206 (dívidas).
OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: GESTÃO DE TERMINAIS
1 - Mostrar todos os terminais
2 - Registar terminal
3 - Operações de terminal
0 - Sair
Escolha uma opção: MENU PRINCIPAL
1 - Abrir
2 - Guardar
3 - Gestão de clientes
4 - Gestão de terminais
5 - Menu de consultas
6 - Ver saldo global
0 - Sair
Escolha uma opção: 